app/test_main.py

```diff
@@ -1,3 +1,36 @@
+import pytest
+
 from app.main import get_human_age
+from typing import List
```

Comment:
Imports from standard library should come first, then third-party packages, and then local imports. All categories should be separated by an extra newline.